lobbyboy/config.py

```diff
@@ -5,43 +5,47 @@
 from dataclasses import dataclass, replace, field, fields, asdict
 from json import JSONDecodeError
 from pathlib import Path
-from typing import Dict, List, Tuple, OrderedDict as typeOrderedDict
+from typing import Dict, List, Tuple, OrderedDict as typeOrderedDict, Optional
 
 import toml
 
-from lobbyboy.utils import lb_dict_factory
+from lobbyboy.exceptions import InvalidConfigException
+from lobbyboy.utils import lb_dict_factory, confirm_dc_type
 
 logger = logging.getLogger(__name__)
 
 
 @dataclass
 class LBConfigUser:
-    authorized_keys: str = ""
-    password: bool = False
+    authorized_keys: str = None
+    password: bool = None
 
     def auth_key_pairs(self) -> List[Tuple]:
         """
 
         Returns:
             tuple: (key_type, key_data)
         """
+        if not self.authorized_keys:
+            return []
         return [tuple(ssh_key.split(maxsplit=1)) for ssh_key in self.authorized_keys.split("\n") if ssh_key]
 
 
 @dataclass
 class LBConfigProvider:
-    load_module: str
-    min_life_to_live: str = "50m"  # 最少存活时间
-    bill_time_unit: str = "55m"
-    private_key: str = "auto"
+    load_module: str = None
+    min_life_to_live: str = None
+    bill_time_unit: str = None
+    # FIXME un used?
+    private_key: str = None
```

Comment:
OH i think it was my mistake, this field may conflict with `extra_ssh_keys`

I will delete it later.